docs/topics/Permissions.md

```diff
@@ -62,8 +62,8 @@ Below is a table of all current permissions, their integer values in hexadecimal
 | USE_SLASH_COMMANDS    | `0x0080000000` `(1 << 31)` | Allows members to use slash commands in text channels                                                                              | T            |
 | REQUEST_TO_SPEAK      | `0x0100000000` `(1 << 32)` | Allows for requesting to speak in stage channels. (_This permission is under active development and may be changed or removed._)   | S            |
 | MANAGE_THREADS \*     | `0x0400000000` `(1 << 34)` | Allows for deleting and archiving threads, and viewing all private threads                                                         | T            |
-| USE_PUBLIC_THREADS    | `0x0800000000` `(1 << 35)` | Allows for creating and participating in threads                                                                                   | T            |
-| USE_PRIVATE_THREADS   | `0x1000000000` `(1 << 36)` | Allows for creating and participating in private threads                                                                           | T            |
+| USE_GUILD_PUBLIC_THREADS    | `0x0800000000` `(1 << 35)` | Allows for creating and participating in threads                                                                                   | T            |
+| USE_GUILD_PRIVATE_THREADS   | `0x1000000000` `(1 << 36)` | Allows for creating and participating in private threads                                                                           | T            |
```

Comment:
i don't think the permission names should be changed; it's inconsistent here, and all permissions are for guilds anyway

Reply:
whoops